felix/bpf-gpl/log.h

```diff
@@ -27,11 +27,13 @@
 		bpf_trace_printk(fmt, sizeof(fmt), ## __VA_ARGS__); \
 } while (0)
 
-#if !defined(UNITTEST) && !defined(__BPFTOOL_LOADER__) && ! (CALI_F_XDP) && !CALI_F_CGROUP
-extern const volatile struct cali_tc_globals __globals;
-#define CALI_TC_LOG(fmt, ...) CALI_LOG("%s" fmt, __globals.iface_name, ## __VA_ARGS__)
+#if ! (CALI_F_XDP) && !CALI_F_CGROUP
```

Comment:
Nit: should this either be `!CALI_F_XDP && !CALI_F_CGROUP` i.e. no space or `!(CALI_F_XDP) && !(CALI_F_CGROUP)`